### ISSUE TITLE
Issue: Current page not be reset.

### ISSUE THREAD

**Issue by hirenkeraliya** [ratiw/vuetable-2#546]
When i changes the per page records then the pagination display the old data like.

If i going to page number 4 and then set the par page is 25 then the pagination current page is to be reset but in real not reset the pagination pages that display current page is 4.

Makes the relation between vuwtable-2 and vuetable-pagination.

**Comment by ratiw** [comment]
@hirenkeraliya Have you check out [this example code](https://codesandbox.io/s/845nzwzk50)?

**Comment by hirenkeraliya** [comment]
@ratiw  Yes i will check this example but this example has also be the same issue like this.

**Comment by acanthis** [comment]
> @ratiw Yes i will check this example but this example has also be the same issue like this.

did you manage to solve this problem?
